src/components/screens/Dashboard.js

```diff
@@ -30,53 +29,61 @@ class Dashboard extends React.Component {
 		return(
 			<View style={styles.content}>
 				<StatusBar translucent={true}
-					backgroundColor={blueColor} />
+					backgroundColor={blueColor} 
+				/>
```

Comment:
Put the closing bracket on the same line of the last prop (we forgot to add it in the coding convention)